zulip_bots/README.md

```diff
@@ -107,9 +107,11 @@ Each bot library simply needs to do the following:
 
 - Define a class that supports the methods `usage`
 and `handle_message`.
+- Optionally add a class member `ACCEPT_EMPTY_MESSAGES`, to
+specify that empty messages will be passed to the bot.
```

Comment:
If these commits go through, rather than the other, then I'll look into that; I wasn't sure if one was outdated, or both should be updated.